imports/ui_3/views/pages/community-page.js

```diff
@@ -552,6 +547,18 @@ Template.Community_page.events({
       singleMethodArgument: true,
     });
   },
+  'click #leaderships .js-period'(event) {
```

Comment:
Hmm. Valahogy ezt az akciot is kell a behaviour-hoz generalni.
Kulonben minden collectionnek, amin van activeperiod behaviour le kell kopoizni ezt, hogy lehessen allitani rajta datumokat.